examples/moe_dualpipe/dualpipe/__init__.py

```diff
@@ -0,0 +1,17 @@
+__version__ = "1.0.0"
```

Comment:
There are dualpipe codes, no need to be included.
In the README.md, explain how to clone the dualpipe codes, setup PYTHONPATH.